src/test/java/org/mockito/internal/creation/bytebuddy/InlineByteBuddyMockMakerTest.java

```diff
@@ -225,6 +225,7 @@ public void is_type_mockable_give_allow_final_mockable_from_JDK() {
     @Test
     public void test_parameters_retention() throws Exception {
         assumeTrue(ClassFileVersion.ofThisVm().isAtLeast(ClassFileVersion.JAVA_V8));
+        assumeTrue(ClassFileVersion.ofThisVm().isLessThan(ClassFileVersion.JAVA_V9)); // Change when ByteBuddy has ASM6
```

Comment:
Yes intentional, alternate way could be to write : 

```java
assumeTrue(ClassFileVersion.ofThisVm().isAtLeast(ClassFileVersion.JAVA_V8) 
        && ClassFileVersion.ofThisVm().isLessThan(ClassFileVersion.JAVA_V9));
```

> Could you also by the way mention #788 in a comment (twice).

Yes I can, but I'm not sure what do you mean ? (in the commit comment or gh comment)